lib/java/com/google/android/material/lists/res/values/dimens.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--General dimensions-->
+    <eat-comment />
+    <dimen name="material_list_item_side_padding">16dp</dimen>
+    <dimen name="material_list_item_item_padding">16dp</dimen>
+    <dimen name="material_list_item_medium_visual_size">40dp</dimen>
+    <dimen name="material_list_item_meta_border">28dp</dimen>
+    <dimen name="material_list_item_large_item_padding">32dp</dimen>
+    <dimen name="material_list_item_action_size">24dp</dimen>
+    <dimen name="material_list_item_rectangle_width">100dp</dimen>
+    <dimen name="material_list_item_rectangle_height">56dp</dimen>
+    <dimen name="material_list_item_square_size">40dp</dimen>
+    <dimen name="material_list_item_meta_height">32dp</dimen>
+    <!-- The meta text uses large padding instead of the recommended 28dp to have the meta text better line
+   up with the primary texts in the "two_line_item" and "three_line_item" xml layouts. --><!-- Single-line specific dimensions -->
```

Comment:
What happened here? Why are the comments merged together?

And the `<!-- Single-line specific dimensions -->` portion should have an `eat-comment` tag as well.